doc/dev/a-quick-guide-to-testing.rst

```diff
@@ -139,11 +139,23 @@ review.
 
```

Comment:
To test the command line interface of a script take a look at `tests/test_scripts.py` to learn about the helper functions that exist for running scripts, capturing their output, and tidying up afterwards.

There are helper functions for loading data and generating temporary names in `khmer_tst_utils.py`.

---

Something like this is what I was thinking of for my comment.